### PR TITLE
GET /user V3

### PR DESCRIPTION
**Pull request by remenska**

### CODE REVIEW

File: src/source/reference/get-user.rst [hunk 1]
```diff
@@ -3,6 +3,38 @@
 
 GET /user
 =========
+:ref:`get_user_v3` | :ref:`get_user_v1`
```
Comment (twessling-icas) [inline src/source/reference/get-user.rst:6]: where's V2?

Comment (remenska) [reply src/source/reference/get-user.rst:6]: I was wondering the same :D imho we don't have to publish it, but can add if preferred

Comment (twessling-icas) [reply src/source/reference/get-user.rst:6]: lol, we never had it in the docs?

Comment (remenska) [reply src/source/reference/get-user.rst:6]: nope, only 2.5 years without doc on it :P

Comment (twessling-icas) [reply src/source/reference/get-user.rst:6]: ROFL ok merge it

Comment (remenska) [reply src/source/reference/get-user.rst:6]: I need approval :P ➕

**Review by twessling-icas — APPROVED** [review]

**Pull request merged**